src/github/findStaleBranches.js

```diff
@@ -2,6 +2,6 @@ const fetchBranches = require('./fetchBranches');
 const filterStaleBranches = require('./filterStaleBranches');
 
 module.exports = async ({ owner, repository }) => {
-  const nodes = await fetchBranches({ owner, repository });
-  return filterStaleBranches({ nodes });
+  const { nodes, defaultBranchName } = await fetchBranches({ owner, repository });
+  return filterStaleBranches({ nodes, defaultBranchName });
```

Comment:
In regards to my rough idea above, how about we refactor `filterStaleBranches` to focus on filtering branches by age and create another function like `excludeBranches` that excludes the specified branches, and the default branch? I think it will make it easier to make changes to exclude protected branches in the future. What do you think?

Reply:
hmmmm...the code to filter `excludedBranches` and `defaultBranches` is only one line, would extracting it out to its own module be overkill?

Reply:
I agree to some extent. Maybe, we can refactor when more necessary.